src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2018-2019 Intel Corporation.
```

Comment:
Minor thing but is 2018 accurate?

Reply:
It is kinda, technically.  This work started back in 2018, but it really should just be 2019.  I will fix.